workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/FieldTypes/ListFieldType.cs

```diff
@@ -55,7 +54,7 @@ public string GetDeserializeUpdateString(string fieldInstance, string schemaObje
 
                 scope.Loop("for (var i = 0; i < listSize; i++)", each =>
                 {
-                    each.WriteLine(new []
+                    each.WriteLine(new[]
```

Comment:
Think this is opposite to how we format this in Unity code

Reply:
Related to the DotSettings thing?

Reply:
I literally copy-pasted this from the current unity dotsettings we have

Reply:
Ah maybe I'm confusing it with something else